sycl/doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc

```diff
@@ -85,13 +87,17 @@ class Functor
 
 It is illegal for a kernel or function to call a function with a mismatched sub-group size requirement, and the compiler should produce an error in this case.  The +reqd_sub_group_size+ attribute is not propagated from a device function to callers of the function, and must be specified explicitly when a kernel is defined.
 
+=== Compiler Flags
+
+The +-fsycl-core-sub-group-size+ flag compiles all kernels in the translation unit as though +[[intel::reqd_sub_group_size(core)]]+ was specified.
+
 === Sub-group Queries
 
-Several aspects of sub-group functionality are implementation-defined: the size and number of sub-groups is implementation-defined (and may differ for each kernel); and different devices may make different guarantees with respect to how sub-groups within a work-group are scheduled.  Developers can query these behaviors at a device level and for individual kernels.  The sub-group size for a given combination of kernel and launch configuration is fixed, and guaranteed to be reflected by device and kernel queries.
+Several aspects of sub-group functionality are implementation-defined: the size and number of sub-groups for certain work-group sizes is implementation-defined (and may differ for each kernel); and different devices may make different guarantees with respect to how sub-groups within a work-group are scheduled.  Developers can query these behaviors at a device level and for individual kernels.  The sub-group size for a given combination of kernel and launch configuration is fixed, and guaranteed to be reflected by device and kernel queries.
```

Comment:
I'm not sure whether there are "coding standards" for spec/extensions sources or whether this affects rendering or not, but it would be really good if the source code of the spec was shaped in like 120 symbols per column, at least.

For example this line contains a whole paragraph, GitHub UI struggles to highlight particular difference and I had to search for it manually. Also, such long lines makes it difficult to post suggestions, as I have to modify just a few words within a long line - against must be hard to see what is exactly changed. An example:
```suggestion
Several aspects of sub-group functionality are implementation-defined: the size and number of sub-groups for certain work-group sizes is implementation-defined (and may differ for each kernel); and different devices may make different guarantees with respect to how sub-groups within a work-group are scheduled change.  Developers can query these behaviors at a device level and for individual kernels.  The sub-group size for a given combination of kernel and launch configuration is fixed, and guaranteed to be reflected by device and kernel queries.
```

**Do not apply this suggestion**, it was posted just to show the problem with reviewing.

Definitely not asking to apply this right now even partially - it is better split functional changes and refactoring :) Just a thought to consider

Reply:
That's a really good point, and I'm sorry that this was so difficult to read and review.  I was looking at `git diff --color-words` offline, and didn't consider how GitHub would render the diff.